### PR TITLE
Header & Footer

### PR DESCRIPTION
**Pull request by ZuhRa-HashiMi**
make the header section
make the footer section

### CODE REVIEW

**Review by TadesseDev — COMMENTED** [review]
### Hi @ZuhRa-HashiMi  

### I'm Tadesse Alemayehu your reviewer for this PR. :hand: 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request 🛑🛑
yPkfDFf

You have submitted a project that has linter tests falling. Please make sure all linters are set up and passing before your next review request.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Review by Ofarouq310 — CHANGES_REQUESTED** [review]
# Status: Failed ❌ 
Hi @ZuhRa-HashiMi  ,

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.


## Invalid Code Review Request 🛑🛑

giphy

You have submitted a project that has linter tests falling. Please make sure all linters are set up and passing before your next review request.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

File: src/index.html [hunk 1]
```diff
@@ -0,0 +1,206 @@
...
+                <a href="#">Phone</a>
+                <a href="#">Address</a>
+              </nav>
+            </div>
+          </div>
+          <div class="copyright">
+            <p class="text-sm text-center mt-6 text-gray-500">
+              &copy; 2022. Omar"Faruq" & Zuhra"Hashimi"
+            </p>
+          </div>
```
Comment (Ofarouq310) [inline src/index.html:199]: - [ ] Kindly change the footer's inner text as you have to follow the required wireframe:
 
image

**Review by Ofarouq310 — CHANGES_REQUESTED** [review]
.

File: src/index.html (hunk 1 above)
Comment (Ofarouq310) [inline src/index.html]: - [ ] Kindly change the footer's inner text as you have to follow the required wireframe:
 
image

**Review by Ofarouq310 — CHANGES_REQUESTED** [review]
Hi @ZuhRa-HashiMi  ,

Great job so far!
You are almost there,  just a couple of simple issues we need to discuss together!

## Highlights 💯
No linters errors ✔️
Following Gitflow ✔️
Following HTML, CSS best practices ✔️

## Required Changes ♻️
- [ ] Kindly fix the disappearing icon at the top left of the page to make it visible.

- [ ] Kindly rename your branch name from 'master' and give it any name according to the feature you are implementing, like 'header-footer' as an example.

- _Please, Check the comments under the review._



## Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._

<hr>

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by Ofarouq310 — APPROVED** [review]
# Status: Approved 🎉 ✔️
Hi @ZuhRa-HashiMi  ,

Your milestone is complete! There is nothing else to say other than... it's time to merge it :shipit:
Congratulations! 🎉

giphy

## Highlights
No linter errors. ✔️
Good README file. ✔️
Milestone worked as expected. ✔️


## Optional suggestions
Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

<hr>

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**